File: src/main/java/no/nav/service/pensjon/regler/to/RevurderingYtelse1967Request.java
package no.nav.service.pensjon.regler.to;

import no.nav.domain.pensjon.regler.beregning2011.SisteBeregning1967;
import no.nav.domain.pensjon.regler.grunnlag.InfoPavirkendeYtelse;
import no.nav.domain.pensjon.regler.krav.Kravhode;
import no.nav.domain.pensjon.regler.vedtak.VilkarsVedtak;

import java.util.Date;
import java.util.Vector;

public class RevurderingYtelse1967Request extends ServiceRequest {


    private Date virkFom;
    private Kravhode kravhode;
    private Vector<VilkarsVedtak> vilkarsvedtakListe;
    private SisteBeregning1967 sisteBeregning1967;
    private InfoPavirkendeYtelse infoPavirkendeYtelse;
    private boolean epsMottarPensjon;
    private boolean beregnForsorgingstillegg;
    private boolean beregnInstitusjonsopphold;



}

File: src/main/java/no/nav/pensjon/regler/to/RevurderingYtelse1967Request.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.beregning2011.SisteBeregning1967
import no.nav.pensjon.regler.domain.grunnlag.InfoPavirkendeYtelse
import no.nav.pensjon.regler.domain.krav.Kravhode
import no.nav.pensjon.regler.domain.vedtak.VilkarsVedtak
import java.util.*

class RevurderingYtelse1967Request {
    var virkFom: Date? = null
    var kravhode: Kravhode? = null
    var vilkarsvedtakListe: Vector<VilkarsVedtak> = Vector()
    var sisteBeregning1967: SisteBeregning1967? = null
    var infoPavirkendeYtelse: InfoPavirkendeYtelse? = null
    var epsMottarPensjon = false
    var beregnForsorgingstillegg = false
    var beregnInstitusjonsopphold = false
}